awesomplete.js

```diff
@@ -40,10 +41,7 @@ var _ = function (input, o) {
 
 	// Create necessary elements
 
-	this.container = $.create("div", {
-		className: "awesomplete",
-		around: input
-	});
+	this.container = this.container(input);
```

Comment:
Do note that this means the `this.container()` function is overwritten and not accessible anymore. Are we sure we don't need to access it again?

Reply:
If I am reading the original code correctly, the container element is never re-created during the object life cycle. That's why I've let the function to be overwritten. Nonetheless, I can change the field used for the container function - do you have a preference for the field name?

Thank you for looking at this PR.

Reply:
Nah, I think `container` is a good name. Perhaps we can store the constructor options somewhere? Like `this.options`? Then we still have access to the function, so you can compare it to the default function in the check in `destroy()` instead of the less foolproof way you're checking now.

Reply:
Done. Added `this.options` to hold the original options passed to `Awesomplete` constructor. Checking `this.options.container` in `destroy()` and cleaning up the container element only when `this.options.container` is not set i.e. when container is originally created by `Awesomplete`.